### PR TITLE
feat: added Routing to different pages , and added database

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
I have added the  routing to all  different pages .

### CODE REVIEW

**Review by sumitk-singh — APPROVED** [review]
-clean code
-looks in structured way.
Good to merge

File: src/App.js [hunk 1]
```diff
@@ -13,7 +14,9 @@ function App() {
           <Route path="/" exact element={ <Home /> }/> 
           <Route path="/Quiz" exact element={ <Quiz /> }/>
           <Route path="/Result" exact element={ <Result /> }/>
-          <Route path="/Guide" exact element={ <Guide /> }/>
+          <Route path="/guideline"  element={ <Guide /> }>
+            <Route path=":guidelineId" element = {<PlayNowQuiz/>}/>
```
Comment (arjun-computer-geek) [inline src/App.js:18]: Add a 404 not found page. It will better

**Review by arjun-computer-geek — APPROVED** [review]
Code is cleaner and readable and well structured.

File: src/App.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/App.js:18]: I will add, thanks for reminding .

**Pull request merged**